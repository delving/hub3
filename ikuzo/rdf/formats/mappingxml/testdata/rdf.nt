<https://klek.si/pqx31b/01339n> <http://purl.org/dc/elements/1.1/identifier> <https://klek.si/pqx31b/01339n> .
<https://klek.si/pqx31b/01339n> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/CreativeWork> .
<https://klek.si/pqx31b/01339n> <http://www.w3.org/2004/02/skos/core#prefLabel> "Glazen buisje"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/dateCreated> "XXc - XXd"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/dateCreated> "1950-01-01"^^<http://www.w3.org/2001/XMLSchema#date> .
<https://klek.si/pqx31b/01339n> <https://schema.org/dateCreated> "1990-01-01"^^<http://www.w3.org/2001/XMLSchema#date> .
<https://klek.si/pqx31b/01339n> <https://schema.org/dateCreated> "2023-02-03T14:47:27.151Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
<https://klek.si/pqx31b/01339n> <https://schema.org/dateModified> "2024-02-01T12:22:04.218Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
<https://klek.si/pqx31b/01339n> <https://schema.org/description> "Glazen buisje - 2e helft van 20e eeuw, glas, afmetingen: restlengte 21 cm, diameter 0,5 cm, laboratoriumglas, aan twee zijden afgebroken<br/>"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/identifier> <https://klek.si/pqx31b/01339n> .
<https://klek.si/pqx31b/01339n> <https://schema.org/image> <https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> .
<https://klek.si/pqx31b/01339n> <https://schema.org/itemCondition> "slecht aan twee zijden afgebroken"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/keywords> "interieur"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/keywords> "laboratorium"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/keywords> "natuurkunde"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/keywords> "natuurwetenschap"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/keywords> "onderwijs"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/keywords> "scheikunde"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/material> "glas"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/name> "Glazen buisje"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/size> "diameter 0.5cm"@nl .
<https://klek.si/pqx31b/01339n> <https://schema.org/size> "hoogte 21cm resthoogte"@nl .
<https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/ImageObject> .
<https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> <https://schema.org/contentSize> "8257" .
<https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> <https://schema.org/contentUrl> <https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> .
<https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> <https://schema.org/encodingFormat> "image/jpeg" .
<https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> <https://schema.org/thumbnailUrl> <https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/thumbnail> .
<https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original> <https://schema.org/uploadDate> "2023-04-12T21:42:47.621Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
