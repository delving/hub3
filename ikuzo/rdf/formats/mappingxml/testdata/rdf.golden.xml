<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:sdo="https://schema.org/" xmlns:skos="http://www.w3.org/2004/02/skos/core#">
    <sdo:CreativeWork rdf:about="https://klek.si/pqx31b/01339n">
        <dc:identifier rdf:resource="https://klek.si/pqx31b/01339n"/>
        <skos:prefLabel xml:lang="nl">Glazen buisje</skos:prefLabel>
        <sdo:dateCreated rdf:dataType="http://www.w3.org/2001/XMLSchema#date">1950-01-01</sdo:dateCreated>
        <sdo:dateCreated rdf:dataType="http://www.w3.org/2001/XMLSchema#date">1990-01-01</sdo:dateCreated>
        <sdo:dateCreated rdf:dataType="http://www.w3.org/2001/XMLSchema#dateTime">2023-02-03T14:47:27.151Z</sdo:dateCreated>
        <sdo:dateCreated xml:lang="nl">XXc - XXd</sdo:dateCreated>
        <sdo:dateModified rdf:dataType="http://www.w3.org/2001/XMLSchema#dateTime">2024-02-01T12:22:04.218Z</sdo:dateModified>
        <sdo:description xml:lang="nl">Glazen buisje - 2e helft van 20e eeuw, glas, afmetingen: restlengte 21 cm, diameter 0,5 cm, laboratoriumglas, aan twee zijden afgebroken&lt;br/&gt;</sdo:description>
        <sdo:identifier rdf:resource="https://klek.si/pqx31b/01339n"/>
        <sdo:image>
            <sdo:ImageObject rdf:about="https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original">
                <sdo:contentSize>8257</sdo:contentSize>
                <sdo:contentUrl rdf:resource="https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/original"/>
                <sdo:encodingFormat>image/jpeg</sdo:encodingFormat>
                <sdo:thumbnailUrl rdf:resource="https://eu.cdn.kleksi.com/pqx31b/crRVxdqX6NGjA1GjsTZ2cCfAn90SK3AoLfkyN5qgU6_ZaBU43yRZ76uQCl775Mw_/thumbnail"/>
                <sdo:uploadDate rdf:dataType="http://www.w3.org/2001/XMLSchema#dateTime">2023-04-12T21:42:47.621Z</sdo:uploadDate>
            </sdo:ImageObject>
        </sdo:image>
        <sdo:itemCondition xml:lang="nl">slecht aan twee zijden afgebroken</sdo:itemCondition>
        <sdo:keywords xml:lang="nl">interieur</sdo:keywords>
        <sdo:keywords xml:lang="nl">laboratorium</sdo:keywords>
        <sdo:keywords xml:lang="nl">natuurkunde</sdo:keywords>
        <sdo:keywords xml:lang="nl">natuurwetenschap</sdo:keywords>
        <sdo:keywords xml:lang="nl">onderwijs</sdo:keywords>
        <sdo:keywords xml:lang="nl">scheikunde</sdo:keywords>
        <sdo:material xml:lang="nl">glas</sdo:material>
        <sdo:name xml:lang="nl">Glazen buisje</sdo:name>
        <sdo:size xml:lang="nl">diameter 0.5cm</sdo:size>
        <sdo:size xml:lang="nl">hoogte 21cm resthoogte</sdo:size>
    </sdo:CreativeWork>
</rdf:RDF>

